lib/api/fields/types/image.php

```diff
@@ -84,35 +84,48 @@ function _beans_get_image_id_attributes( $id, array $field, $is_multiple ) {
 }
 
 /**
- * Get the image's URL and alt values.
+ * Get the image's URL.
  *
  * @since 1.5.0
  *
  * @param mixed $image_id The image's attachment ID.
  *
- * @return array|void
+ * @return string|void
  */
-function _beans_get_image_url_and_alt( $image_id ) {
+function _beans_get_image_url( $image_id ) {
 	$image_id = (int) $image_id;
 
 	// If this is not a valid image ID, bail out.
 	if ( $image_id < 1 ) {
 		return;
 	}
 
-	$image_url = beans_get( 0, wp_get_attachment_image_src( $image_id, 'thumbnail' ) );
+	return beans_get( 0, wp_get_attachment_image_src( $image_id, 'thumbnail' ) );
+}
+
+/**
+ * Get the image's alt description.
+ *
+ * @since 1.5.0
+ *
+ * @param mixed $image_id The image's attachment ID.
+ *
+ * @return string|void
+ */
+function _beans_get_image_alt( $image_id ) {
```

Comment:
Unit (monkey-patch `get_post_meta()`) or system tests (assign to post_meta and actually read from it)?

Reply:
Added tests in the latest commit.